makefile_components/base_build_go.mak

```diff
@@ -93,6 +93,11 @@ golint:
                  	    $(BUILD_IMAGE)                                                     \
                  	    bash -c 'golint $(SRC_AND_UNDER)'
 
+release: build
```

Comment:
I'd rather add this in the very few Makefiles that actually need it. It's unusual for it to be needed.